Javatrix.java

```diff
@@ -481,5 +481,27 @@ public Javatrix times(double scalar)
         }
         return actual;
     }
+
+    /**
+     * Matrix transpose.
+     *
+     * @return A'
+     */
+    public Javatrix transpose()
+    {
+        int rows = this.getColumnDimension();
```

Comment:
I intentionally did not use m and n here because those will already be set for the calling element. I also wanted to be able to switch the calling object rows with the returned object columns, so the matrix is actually transposed, and I didn't want it to be confusing by using m and n.